src/WebJobs.Script.WebHost/Security/BlobStorageSecretsRepository.cs

```diff
@@ -182,6 +188,7 @@ private void Dispose(bool disposing)
         public void Dispose()
         {
             Dispose(true);
+            GC.SuppressFinalize(this);
```

Comment:
This is unrelated, but for some reason the project wouldn't build without this change. This seems to match other Dispose patterns, though.

Reply:
Dispose pattern in dotnet requires that  https://docs.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-dispose

Reply:
oh that's a nice resource, thanks!